taipy/core/data/generic.py

```diff
@@ -108,7 +108,7 @@ def __init__(
             editor_expiration_date,
             **properties,
         )
-        if not self._last_edit_date:
+        if not self._last_edit_date:  # type: ignore
```

Comment:
WHy do we need to add an ignore here? Can't we avoid it?

Reply:
mypy couldn't detect the type of `self._last_edit_date` so it throws an error :/